### ISSUE TITLE
New Zim File for Bibe Books

### ISSUE THREAD

**Issue by RavanJAltaie** [openzim/zim-requests#640]
As a part of scouting Grey Box content, we need to create zim file for the details below:

Website URL: https://www.bibebook.com/
License: Creative Commons
Desired ZIM Title: Bibe Books
Desired ZIM Description: 1700 ebooks gratuits
Desired ZIM Icon –png (URL or attach one):
Language (ISO 639-3): fra
Is this a MediaWiki?: no

**Comment by RavanJAltaie** [comment]
Recipe Created 
https://farm.openzim.org/recipes/bibebooks_fr_all

**Comment by RavanJAltaie** [comment]
@rgaudin the scraper here started since 10 days ago, is than normal? 
Shall I wait more?

**Comment by rgaudin** [comment]
You should take a look at the website before creating the recipe.

Search engines should be excluded always (except for client-side search engine) because they generate tons of useless, mostly duplicated pages.

**Comment by RavanJAltaie** [comment]
@rgaudin you are correct, it didn't start for 20 days, it will not work out with this scraper.
I'll keep an eye for any websites with search engine.

**Comment by benoit74** [comment]
Custom scraper seems to be pretty easy to write, the website looks not updated anymore.

If one goes to http://www.bibebook.com/files/download/zip/packs_classiques/, it is visible the packs have not been updated since 2016.

The whole catalog is available at http://www.bibebook.com/files/download/catalogues/Bibebook-library.uni in XML format.

Scraper can hence simply:
- download this catalog
- use this to download pictures, pdf, epub and mobi versions of the books
- push everything to a ZIM

Good way to learn how to write a scraper tbh.
